spec/support/rails_template_with_data.rb

```diff
@@ -1,22 +1,145 @@
 apply File.expand_path("../rails_template.rb", __FILE__)
 
-%w{Post User Category}.each do |type|
+inject_into_file 'config/initializers/active_admin.rb', <<-RUBY, after: "ActiveAdmin.setup do |config|\n"
+
+  config.comments_menu = { parent: 'Administrative' }
+RUBY
+
+inject_into_file 'app/admin/admin_user.rb', <<-RUBY, after: "ActiveAdmin.register AdminUser do\n"
+
+  menu parent: "Administrative", priority: 1
+RUBY
+
+generate :'active_admin:page', 'KitchenSink'
+
+gsub_file 'app/admin/kitchen_sink.rb', /^ *content.*?end/m, <<-RUBY
+
+  sidebar "Sample Sidebar" do
+    para "Sidebars can also be used on custom pages."
+    para do
+      a "Active Admin", href: "https://github.com/activeadmin/activeadmin"
+      text_node "is a Ruby on Rails framework for"
+      em "creating elegant backends"
+      text_node "for"
+      strong "website administration."
+    end
+    para do
+      abbr "HTML", title: "HyperText Markup Language"
+      text_node "is the most basic building block of the Web."
+    end
+  end
+
+  content do
+    columns do
+      column do
+        panel "Panel title" do
+          h1 "This is an h1"
+          h2 "This is an h2"
+          h3 "This is an h3"
+        end
+      end
+      column do
+        table_for User.all do
+          column :id
+          column :display_name
+          column :username
+          column :age
+          column :updated_at
+        end
+      end
+    end
+
+    tabs do
+      tab :first do
+        ul do
+          li "List item"
+          li "Another list item"
+          li "Last item"
+        end
+        ol do
+          li "First list item"
+          li "Second list item"
+          li "Third list item"
+        end
+      end
+      tab :second do
+        para "A popular quote."
+        blockquote do
+          text_node "\"Be yourself; everyone else is already taken.\""
+          cite "― Oscar Wilde"
+        end
+      end
+      tab :third do
+        para "Third tab content."
+      end
+    end
+  end
+RUBY
+
+%w{Post User Category Tag}.each do |type|
   generate :'active_admin:resource', type
 end
 
 inject_into_file 'app/admin/category.rb', <<-RUBY, after: "ActiveAdmin.register Category do\n"
 
+  config.create_another = true
+
   permit_params [:name, :description]
```

Comment:
@javierjulio , I think ```permit_params``` are declared
like
```
permit_params :name, :description
```

however this works with array too, so not a blocker